amical/data_processing.py

```diff
@@ -483,6 +483,9 @@ def select_clean_data(
     verbose=False,
     ihdu=0,
     display=False,
+    *,
+    show_bad_removed=True,
+    show_nframe=0,
```

Comment:
I see no reason not to keep the existing names. Making them different would actually hurt the kind of usage you are encouraging here.
Please make sure that the default values match the ones from `show_clean_params`
```suggestion
    remove_bad=True,
    nframe=0,
```

Reply:
Good point! The default values do match the `show_clean_params` ones.